app/services/ResponsiblePeopleService.scala

```diff
@@ -38,9 +39,9 @@ class ResponsiblePeopleService @Inject()(val dataCacheConnector: DataCacheConnec
   def updateFitAndProperFlag(responsiblePeople: Seq[ResponsiblePerson], indices: Set[Int]): Seq[ResponsiblePerson] =
     responsiblePeople.zipWithIndex.map { case (rp, index) =>
       val updated = if (indices contains index) {
-        rp.hasAlreadyPassedFitAndProper(Some(true))
+        rp.approvalFlags(ApprovalFlags(hasAlreadyPassedFitAndProper = Some(true)))
```

Comment:
this needs to be a copy so that the approvals isn't wiped.

so it would be something like rp.approvalFlags(rp.approvalFlags.copy(hasAlreadyPassedFitAndProper = Some(true)